src/strategyDP/SecondAC.java

```diff
@@ -0,0 +1,20 @@
+package strategyDP;
+
+import java.util.Scanner;
+
+public class SecondAC implements BookTicketStratergy {
```

Comment:
I am not sure if you understood this pattern,
you are demonstrating different classes with same bookTicket Method